instrumentation/kamon-akka/src/common/scala/kamon/instrumentation/akka/instrumentations/ActorInstrumentation.scala

```diff
@@ -28,6 +28,9 @@ import kanela.agent.libs.net.bytebuddy.asm.Advice.{Argument, OnMethodEnter, OnMe
 
 class ActorInstrumentation extends InstrumentationBuilder {
 
+  onType("akka.actor.dungeon.Dispatch")
```

Comment:
I think I've caused a regression with Akka 2.5 with this one. It looks like this advice will need to be split between Akka versions